fedora_elections/elections.py

```diff
@@ -144,11 +141,19 @@ def vote_range(election):
         max_range=max_selection)
 
     if form.validate_on_submit():
-
         if form.action.data == 'submit':
             for candidate in form:
                 if candidate.short_name in ['csrf_token', 'action']:
                     continue
+                if revote:
+                    vote = update(models.Vote).\
+                    where(models.Vote.candidate_id == candidate.short_name
+                    and models.Vote.voter == flask.g.fas_user.username
+                    and models.Vote.election == election.id).\
+                    values(value = candidate.data)
+                    SESSION.execute(vote)
+                    #break out of candidate loop
+                    continue
```

Comment:
The only way I know how to do it using the ORM does not use the where clause so I had to create a for loop inside the outer for loop and it is inefficient to do it that way, how can I convert this code to ORM, I am completely for this change :)